airflow/providers/snowflake/CHANGELOG.rst

```diff
@@ -24,6 +24,28 @@
 Changelog
 ---------
 
+3.4.0
```

Comment:
Should be 4.0.0 ( there is 5.0 entry in log but i guess it meant to ve 4.0)

Reply:
Yep.

Reply:
Yep.